Bugzilla/CPAN.pm

```diff
@@ -104,6 +104,16 @@ sub feature {
     return $FEATURE_LOADED{$feature_name} = 1;
 }
 
+sub preload_features {
+    my ($class) = @_;
+    return 0 unless _CAN_HAS_FEATURE;
```

Comment:
pretty much I've never been able to measure a difference from using variables or not, it seems like just method calls / function calls are slow.